src/Exception/GuzzleException.php

```diff
@@ -1,13 +1,20 @@
 <?php
 namespace GuzzleHttp\Exception;
 
-/**
- * @method string getMessage()
- * @method \Throwable|null getPrevious()
- * @method mixed getCode()
- * @method string getFile()
- * @method int getLine()
- * @method array getTrace()
- * @method string getTraceAsString()
- */
-interface GuzzleException {}
+use Throwable;
+
+if (interface_exists(Throwable::class)) {
```

Comment:
`interface_exists` has a second parameter `autoload` that defaults to true: https://www.php.net/manual/en/function.interface-exists.php

Can we explicitely pass this as `false`? It makes no sense imo to try autoloading a SPL class/interface, and causes issues with bad autoloader implementations that `include`/`require` everything passed to them even when such a file does not exist.

I can make a PR if needed.

Reply:
pr welcome